CHANGELOG.md

```diff
@@ -8,6 +8,138 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+
+### Added
+- `Generic.PHP.DiscourageGoto`, `Generic.PHP.LowerCaseType`, `Generic.WhiteSpace.ArbitraryParenthesesSpacing` and `PSR12.Keywords.ShortFormTypeKeywords` to the `WordPress-Core` ruleset.
+- Checking the spacing around the `instanceof` operator to the `WordPress.WhiteSpace.OperatorSpacing` sniff.
+
+### Changed
+- The minimum required `PHP_CodeSniffer` version to 3.3.1 (was 2.9.0).
+- The namespace used by WordPressCS has been changed from `WordPress` to `WordPressCS\WordPress`.
+    This was not possible while `PHP_CodeSniffer` 2.x was still supported, but WordPressCS, as a good Open Source citizen, does not want to occupy the `WordPress` namespace and is releasing its use of it now this is viable.
+- The `WordPress.DB.PreparedSQL` sniff used the same error code for two different errors.
+    The `NotPrepared` error code remains, however an additional `InterpolatedNotPrepared` error code has been added for the second error.
+    If you are referencing the old error code in a ruleset XML file or in inline annotations, you may need to update it.
+- The `WordPress.NamingConventions.PrefixAllGlobals` sniff used the same error code for some errors as well as warnings.
+    The `NonPrefixedConstantFound` error code remains for the related error, but the warning will now use the new `VariableConstantNameFound` error code.
+	The `NonPrefixedHooknameFound` error code remains for the related error, but the warning will now use the new `DynamicHooknameFound` error code.
+    If you are referencing the old error codes in a ruleset XML file or in inline annotations, you may need to update these to use the new codes instead.
+- The `WordPress.Security.NonceVerification.NoNonceVerification` sniff used the same error code for both an error as well as a warning.
+    The old error code `NoNonceVerification` is no longer used.
+    The `error` now uses the `Missing` error code, while the `warning` now uses the `Recommended` error code.
+    If you are referencing the old error code in a ruleset XML file or in inline annotations, please update these to use the new codes instead.
+- The `WordPress.WP.DiscouragedConstants` sniff used to have two error codes `UsageFound` and `DeclarationFound`.
+    These error codes will now be prefixed by the name of the constant found to allow for more fine-grained excluding/ignoring of warnings generated by this sniff.
+    If you are referencing the old error codes in a ruleset XML file or in inline annotations, you may need to update these to use the new codes instead.
+- The `WordPress.WP.GlobalVariablesOverride.OverrideProhibited` error code has been replaced with the `WordPress.WP.GlobalVariablesOverride.Prohibited` error code.
```

Comment:
The preposition by seems more appropriate in this context, (replaced by) instead of (replaced with).